silx/gui/plot/ProfileMainWindow.py

```diff
@@ -56,7 +58,10 @@ class ProfileMainWindow(qt.QMainWindow):
 
     sigProfileMethodChanged = qt.Signal(str)
     """Emitted when the method to compute the profile changed (for now can be
-    sum or mean)"""
+    sum or mean)
+
+    Note: This signal should be removed.
```

Comment:
Then we can first remove the docstring and put a comment

Reply:
unless we consider `ProfileMainWindow` not being part of the public API.

Reply:
We could create a deprecated signals, with a deprecating property method returning the signal.

Reply:
But i would prefer to talk about it in the next PR.

Reply:
OK